features/old/accounts/buyers/account_fields.feature

```diff
@@ -46,6 +46,18 @@ Feature: Buyer side, account fields
        And I press "Update"
      Then I should see "The account information was updated."
 
+  Scenario: Read-only extra fields are ignored on update
+    Given I log in as "bob" on foo.3scale.localhost
+    And provider "foo.3scale.localhost" has the field "non_editable" for accounts as editable
+    Then I go to the account edit page
+    And provider "foo.3scale.localhost" has the field "non_editable" for accounts as read only
```

Comment:
So to hack the form fists field is ediable and then made non-editable and submitting the form as if it was editable. Ok..

Reply:
Yeah, I'm a genius, you're welcome